contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1107,9 +1111,11 @@ select cast(t.typname as text) as name
     ELSE  c.collname 
     END as collation_name
   , case when tt.typrelid is not null then 0
-         else case when typnotnull then 0 else 1 end
-    end
-    as is_nullable
+        else CASE t.typbasetype
```

Comment:
Why are we not doing the same thing as above here?

```
CASE ti.tsql_type_name
      WHEN 'timestamp' THEN 0
      WHEN 'sysname' THEN 0
    ELSE case when typnotnull then 0 else 1 end
    end as is_nullable

Reply:
We cannot use ti.tsql_type_name to compare here as this is taken from sys.babelfish_typecode_list() which contains only system data types names which will be  different from user defined data type names  , so I used the system_type_id to identify which system data type it is created from and if that system_type_id matches that of  timestamp and sysname system_type_id then it should return 0.